src/derived/grid8.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
That's a good point, we could simply just export this updated `<Base>` and no need to add an additional component. I'll update